imports/api/shareddocs/shareddocs.js

```diff
@@ -17,6 +17,15 @@ Shareddocs.hasPermissionToUpload = function hasPermissionToUpload(userId, doc) {
   else return user.hasPermission('shareddocs.upload', doc);
 };
 
+Shareddocs.hasPermissionToUpdate = function hasPermissionToUpdate(userId, doc) {
```

Comment:
Ez a nagy blokk kod most szuletett, vagy masolata a masik fuggvenynek?
Azert nem celszeru ilyen hosszu fv-eket lemasolni, mert kesobb ha meg akarom tudni hogy ez ugyanaz e mint a masik, akkor 15 sort kell szemmel vegig osszehaosnlitanom, ami hosszu perceket vesz igenybe, hogy megkapjam erre a valaszt. Ilyenkor celszeru a fv testet elnevezni valahogy es mindket helyrol meghivni.
Az editphoto eseteben is, az OK, hogy legyen erre kulon action, de akkor tegyen csak annyit hogy meghvja az edit action-nek a run-jat, es akkor latszik hogy van egy ilyen action is, de ugyanazt csinalja mint az edit.

Reply:
Sharedocs.permissionből már volt ott kettő, gondoltam, volt rá ok, hogy külön vannak (bármi is legyen az). De azokat akkor összevonom.
Az editphotot meg fönt leírtam, hogy miért nem lehet az edit action-nek run-ja. Kicsit bonyolultabbnak tűnt megoldani, hogy az action-buttonra kattintáskor ne csinálja azt, amit az action buttonok csinálnak, hanem inkább azt, amit most külön kifejeztünk a templét js-edit eventjében. Mert ott még nem akarjuk, hogy előugorjon a komment szerkesztő autoform modál. A modált csak akkor akarjuk, ha fotót is szeretne csatolni, és megnyomja azt a gombot.

Reply:
nem a shareddocs.permission-re gondoltam, hanem a 20.-27. sorig tarto kodra, ami sok if-en megy vegig hogy eldontse van e permission.
Ez uj, vagy valahonnan ide lett kopizva?
Mert ha ketszer van most leirva ez a bonyolult if-es logika, az problemas lesz, amikor bele kell javitani.

Reply:
Nekem zolden latszik a PR diff-ben, es mashol nem piros, szoval nem atkerult valahonnan, hanem megjelent most itt. De nem ugy nez ki mintha most irtad volna, ezert tunik ugy mintha kopia lenne